Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2688,6 +2688,25 @@ def test_get_context(self):
         finally:
             loop.close()
 
+    def test_proper_refcounts(self):
+        class Break:
+            def __str__(self):
+                raise Exception("break")
+        
+        obj = object()
+        initial_refcount = sys.getrefcount(obj)
+
+        coro = coroutine_function()
+        task = asyncio.Task.__new__(asyncio.Task)
+
+        for _ in range(10000):
```

Comment:
Do we really need 10000 iterations to find a leak?

Reply:
Well, maybe not 10000, but finding leaks is certainly easier with more iterations. Maybe 1000 should do?

Reply:
I changed it to 5, the bug would be triggered every time so with a low amount you should still be able to see the ref leak.